.ci/travis/travis.sh

```diff
@@ -174,8 +174,8 @@ assembly-run-all-jar)
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo version:$CS_POM_VERSION
   mkdir -p .ci-temp
-  FOLDER=src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks
-  FILE=InputRightCurlyAloneOrEmptyNoViolations.java
+  FOLDER=src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing
+  FILE=InputCustomImportOrderNoImports.java
```

Comment:
@romani 
https://github.com/checkstyle/checkstyle/pull/8030#issuecomment-609428376
https://github.com/checkstyle/checkstyle/pull/8030#issuecomment-614720431
Can we come to some type of agreement on what this validation should be doing and keep it consistent. It seems pointless if we keep switching from file to file on what it will validate.

Reply:
yes we can, all we need is to create simple Input file, it will pass "always" ....

Reply:
I made a very simple example as part of #8105. Hopefully that will be the last time we ever need to change that file.